sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -96,13 +112,13 @@ def test_y_multivariate_raises_error(Forecaster):
     """Test that wrong y scitype raises error (uni/multivariate if not supported)."""
     f = _construct_instance(Forecaster)
 
-    if f.get_tag("scitype:y") == "univariate":
+    if f.get_tag("scitype:y") in ["univariate"]:
```

Comment:
should this not be `["univariate", "both"]`?

Reply:
For this situation, I did not check for "both". I made sure to pass if both